<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>1040</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Cahier de jeu | Rugby</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/img/ballon.png</normaloff>:/img/ballon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>-10</y>
      <width>241</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="testWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>239</width>
       <height>659</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="create_Trajectory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Crée une trajectoire</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modify_trajectory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier une trajectoire</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ouvrir graphic scene</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Vitesserouge">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Courir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Vitesse_orange">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Trottiner</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vitesse_vert">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Marcher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>tester conversion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_anim">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spin_depChang">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="traj_name_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Trajectoire sélectionner</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="distanceTrajectoire">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Distance de la trajectoire :</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>19</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionImport"/>
    <addaction name="actionExporte"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="actionNouveau">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Importer depuis une base de donnée</string>
   </property>
  </action>
  <action name="actionExporte">
   <property name="text">
    <string>Exporter vers une base de donnée</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>create_Trajectory</tabstop>
  <tabstop>modify_trajectory</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Vitesserouge</tabstop>
  <tabstop>Vitesse_orange</tabstop>
  <tabstop>vitesse_vert</tabstop>
  <tabstop>testButton</tabstop>
  <tabstop>start</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
